lib/mozart_fetcher/decoder.ex

```diff
@@ -19,7 +19,7 @@ defmodule MozartFetcher.Decoder do
     case struct!(struct, map) do
       {:error} ->
         ExMetrics.increment("error.components.decode")
-        Stump.log(:error, %{message: "Invalid keys passed to to_struct", struct: struct, map: map})
+        Stump.log(:error, %{message: "Invalid keys passed to to_struct", map: map})
```

Comment:
`struct` here is just an atom, the type of struct that you would want the decoder to return for you. I think it's still safe to Log this as it's not coming from an external source